packages/strapi-design-system/src/Field/FieldInput.js

```diff
@@ -47,7 +47,9 @@ const Input = styled.input`
 `;
 
 export const InputWrapper = styled(Flex)`
-  border: 1px solid ${({ theme, hasError }) => (hasError ? theme.colors.danger600 : theme.colors.neutral200)};
+  border: 1px solid
+    ${({ theme, hasError, showErrorBorder }) =>
```

Comment:
maybe I missed a discussion about it - I was curious about why adding this new `showErrorBorder` prop?

Reply:
Hello, because Hicham suggests me to handle differently how we show errors in the DateTimePicker so I thought it was good to just use a boolean to show only the red border and not the error messages for the DatePicker and TimePicker

Reply:
Why would we not want to show the error message for these fields?

Reply:
because we want to show just the Error message in the DateTimePicker

Reply:
I think the confusion here is you're passing a prop to the component to show an error but normally its handled by the context. It creates two ways of _almost_ doing the same thing.

Maybe the `error` from the context if a `string` should show a message otherwise if it's a `boolean` should just change the styling. 

This would create a more uniform API to handle field errors as opposed to splintering. What do others think?

Reply:
just to be really specific:
- if `error === string`: error message + error styling
- if `error === boolean`: error styling

is that it @joshuaellis ?

Reply:
Yes, exactly @ronronscelestes, that way there is one way to do things 😄

Reply:
I like it 💃

Reply:
I like the idea @joshuaellis

Reply:
I tried to implement @joshuaellis idea, what do you think? https://github.com/strapi/design-system/pull/703/commits/b3361c7a2f09e34a13c38db103818fd48e9b20ff

Reply:
LGTM